docs/config.md

```diff
@@ -396,6 +396,234 @@ which is convenient to conduct various experiments.
 
     ```
 
+### Config System for Spatio-Temporal Action Detection
+
+We incorporate modular design into our config system,
+which is convenient to conduct various experiments.
+
+#### An Example of FastRCNN
+
+To help the users have a basic idea of a complete config structure and the modules in an spatio-temporal action detection system,
+we make brief comments on the config of FastRCNN as the following.
+For more detailed usage and alternative for per parameter in each module, please refer to the API documentation.
+
+```python
+# model setting
+model = dict( # Config of the model
+    type='FastRCNN',  # Type of the detector
+    backbone=dict(  # Dict for backbone
+        type='ResNet3dSlowOnly',  # Name of the backbone
+        depth=50, # Depth of ResNet model
+        pretrained=None,   # The url/site of the pretrained model
+        pretrained2d=False, # If the pretrained model is 2D
```

Comment:
Unrelated to this pr.

`pretrained2d` -> `is_pretrained2d` might be clearer

Reply:
I think this flag can even be removed since when the checkpoint is loaded, you know if it is 2d or 3d (by looking at conv.weight shape, etc.).

Reply:
yeah